### PR TITLE
Improved view for connection tab #8021

### PR DESCRIPTION
**Pull request by vishu1993**

### CODE REVIEW

**Comment by vishu1993** [comment]
ebaychannel

File: view/sale_channel_form.xml [hunk 1]
```diff
@@ -11,9 +11,11 @@
             <label name="is_ebay_sandbox" />
             <field name="is_ebay_sandbox" />
             <newline/>
-            <label name="ebay_token" />
-            <field name="ebay_token" colspan="3" />
-            <button string="Check eBay Token Status" name="check_ebay_token_status"/>
+        </group> 
+        <group id="ebay_tocken" col="2" colspan="4" states="{'invisible': Not(Eval('source') == 'ebay')}" >
```
Comment (tarunbhardwaj) [inline view/sale_channel_form.xml]: typo

**Comment by prakashpp** [comment]
Buttons are taking full width that is not looking good.

**Comment by vishu1993** [comment]
ebay1

**Pull request merged**